src/NuGet.Core/NuGet.Protocol/Plugins/Messages/GetAuthenticationCredentialsRequest.cs

```diff
@@ -29,6 +29,12 @@ public sealed class GetAuthenticationCredentialsRequest
         [JsonRequired]
         public bool IsNonInteractive { get; }
 
+        /// <summary>
+        /// CanPrompt
+        /// </summary>
+        [JsonRequired]
+        public bool CanPrompt { get; }
```

Comment:
Can we have a more suitable name? `CanPrompt` confuses me with `IsNonInteractive` as i mentione d in another comment.